### DISCUSSION TITLE
GitHub is not approving the student document

### DISCUSSION THREAD

**Discussion by hamitarsln** [community/community#118351] [GitHub Education]
### Select Topic Area

Bug

### Body

I was going to get the GitHub Student Developer Pack, but I'm trying to do this as a high school student. It asks for the name of my school, which I provide, and then in the following process, it asks me to take a photo of my student document and upload it.

![image](https://github.com/community/community/assets/98461887/b236ef8d-a9c2-4f6a-9046-5b17314c1e8b)

It asks me to take a photo of my student document and upload it, but since my computer doesn't have a camera, I used OBS Studio's virtual camera system to display my student document in OBS.

![image](https://github.com/community/community/assets/98461887/bd77400f-4daf-4934-b866-e8e11a5bb679)

I selected the 'dated school ID' option as proof and submitted it, but it's not getting approved at all. How can I solve this issue?

**Comment by MarouaneBenbetka** [comment]
Try joining by phone , this way you won't have a problem with camera, 

for me i also don't have a camera in my PC , but i used an application for ASUS devices to link my phone to the pc and than used the camera, if you can't do that just submit using your phone.  

if this fixed your issue mark it as answered, GOOD LUCK .

**Comment by KryukovSA** [comment]
Такая же проблема я пробовал как загружать так и фотографировать. Ничего не помогло. Названия везде совпадают.
